### DISCUSSION TITLE
milvus支持索引参数的动态更新嘛？

### DISCUSSION THREAD

**Discussion by CHENLONG871** [milvus-io/milvus#38491] [Q&A and General discussion]
milvus已基于hnsw构建索引，设置参数M=8，efConstruction=64,并且插入了部分数据，现在的问题是，希望在同一个collection下，改变索引构建参数M=16，再插入新的数据，而历史数据不做修改，这样的操作是否可行？

**Comment by yhmo** [comment]
目前不行。改变索引参数就会对该向量字段重建索引。你调用create_index()修改参数之前，它也会提示你说已有索引存在，要先删掉旧的索引才能建新的索引。

**Comment by CHENLONG871** [comment]
ok, 感谢答复
